src/DnsUtils.test.ts

```diff
@@ -0,0 +1,48 @@
+import DnsUtils from "./DnsUtils";
+import DnsRecordsError from "./errors/dnsRecordsError";
+import { CryptoRecords, DnsRecord, DnsRecordType } from "./publicTypes";
+
+let dnsUtils: DnsUtils;
+beforeAll(() => {
+  dnsUtils = new DnsUtils();
+});
+
+describe('DnsUtils', () => {
+  it('toClassical', () => {
+    const record: CryptoRecords = {
+      "dns.ttl": "128",
+      "dns.A": '["10.0.0.1","10.0.0.2"]',
+      "dns.A.ttl": "90",
+      "dns.AAAA": '["10.0.0.120"]'
+    };
+    const classic: DnsRecord[] = dnsUtils.toList(record);
+    expect(classic).toStrictEqual([
+      { ttl: 90, value: '10.0.0.1', type: 'A' },
+      { ttl: 90, value: '10.0.0.2', type: 'A' },
+      { ttl: 128, value: '10.0.0.120', type: 'AAAA' }
+    ]);
+  });
```

Comment:
Add tests for the following scenarios:

*  when `dns.A` is set to `[]`
* when ttl is not a number